### ISSUE TITLE
Local NGO referral partners list

### ISSUE THREAD

**Issue by SallyMcGrath** [CodeYourFuture/CYF-Staff-Tickets#590]
### Background

I was recently updating [Signposts](https://signposts.codeyourfuture.io/) and this occurred to me.

I parsed the [UEL Directory of Services](https://uel.ac.uk/our-research/research-school-psychology/refugee-mental-health-wellbeing-portal/resource-centre/directory-services), which I have been using recently, with AI to give me this list of regional migrant support services. I'm looking for context on these orgs. Do we know them? Should we know them? I started asking @AmyCYF this in the DLP channel but realised it should probably be a ticket.

### London:

- [ ] Asylum Aid - [www.asylumaid.org.uk](http://www.asylumaid.org.uk/)
- [ ] Art Refuge UK - [www.artrefuge.org.uk](http://www.artrefuge.org.uk/)
- [ ] JAN Trust - [www.jantrust.org](http://www.jantrust.org/)
- [ ] Micro Rainbow - [www.microrainbow.org](http://www.microrainbow.org/)
- [ ] Immigrant Counselling & Psychotherapy (ICAP) - [www.icap.org.uk](http://www.icap.org.uk/)
- [x] Refugee Education UK - [www.reuk.org](http://www.reuk.org/)
- [ ] Room to Heal - [www.roomtoheal.org](http://www.roomtoheal.org/)
- [ ] Network of Eritrean Women - https://www.networkoferitreanwomen-uk.co.uk/

### Glasgow:

- [ ] The Bridges Programmes - [www.bridgesprogrammes.org.uk](http://www.bridgesprogrammes.org.uk/)
- [ ] Hemat Gryffe Women’s Aid - [www.hematgryffe.org.uk](http://www.hematgryffe.org.uk/)
- [ ] Positive Action in Housing - [www.paih.org](http://www.paih.org/)
- [ ] Room For Refugees - [www.roomforrefugees.com](http://www.roomforrefugees.com/) *do we have a referral account?

#### Birmingham

- [ ] Birch Network - [www.birchnetwork.org](http://www.birchnetwork.org/)
- [ ] Refugee Council’s Birmingham Therapeutic Service - [www.refugeecouncil.org.uk/get-support/services/birmingham-therapeutic-service](http://www.refugeecouncil.org.uk/get-support/services/birmingham-therapeutic-service)
- [ ] Lifeline options - [www.lifelineoptions.org.uk](http://www.lifelineoptions.org.uk/)
- [ ] Restore - [www.restore-uk.org](http://www.restore-uk.org/)
- [ ] St. Chad's Sanctuary - [www.stchadssanctuary.com](http://www.stchadssanctuary.com/)

#### Coventry:

- [ ] Coventry Refugee and Migrant Centre - [www.covrefugee.org](http://www.covrefugee.org/)

#### Solihull:

Solihull Welcome - [www.centralsolihullchurches.org.uk/solihull-welcome](http://www.centralsolihullchurches.org.uk/solihull-welcome)

#### Leeds:

- [ ] Leeds Asylum Seekers Support Network (LASSN) - [www.lassn.org.uk](http://www.lassn.org.uk/)
- [ ] Leeds Refugee Education Training Advice Service (RETAS) - [www.retasleeds.weebly.com](http://www.retasleeds.weebly.com/)
- [ ] Leeds Refugee Forum - [www.leedsrefugeeforum.org.uk](http://www.leedsrefugeeforum.org.uk/)
- [ ] Positive Action for Refugees & Asylum Seekers (PAFRAS) - [www.pafras.org.uk](http://www.pafras.org.uk/)
- [ ] Refugee Council’s Health Access for Refugees Programme in South Yorkshire - [www.refugeecouncil.org.uk/get-support/services/health-access-for-refugees](http://www.refugeecouncil.org.uk/get-support/services/health-access-for-refugees)
- [ ] Solace - [www.solace-uk.org.uk](http://www.solace-uk.org.uk/)
- [ ] Touchstone - [www.touchstonesupport.org.uk](http://www.touchstonesupport.org.uk/)
- [ ] Women’s Counselling & Therapy Service - [www.womenstherapyleeds.org.uk](http://www.womenstherapyleeds.org.uk/)

#### Manchester:

- [x] Boaz Trust - [www.boaztrust.org.uk](http://www.boaztrust.org.uk/)
- [ ] Manchester Refugee Support Network (MRSN) - [www.mrsn.org.uk](http://www.mrsn.org.uk/)
- [ ] Rainbow Haven - [www.rainbowhaven.org.uk](http://www.rainbowhaven.org.uk/)

#### Liverpool:

- [ ] Family Refugee Support Project (FRSP) - [www.frsp.org.uk](http://www.frsp.org.uk/)

#### Huddersfield:

- [ ] Destitute Asylum Seekers Huddersfield (DASH) - [www.huddsdash.org](http://www.huddsdash.org/)

#### Sheffield:
- [x] Ashiana Sheffield - [www.ashianasheffield.org](http://www.ashianasheffield.org/)
- [x] Assist Sheffeild - www.assistsheffield.org.uk 
- [x] New Roots Therapeutic Service, Sheffield & South Yorkshire - [www.refugeecouncil.org.uk/get-support/services/new-roots-therapeutic-services-sheffield-and-south-yorkshire](http://www.refugeecouncil.org.uk/get-support/services/new-roots-therapeutic-services-sheffield-and-south-yorkshire)
- [ x] City of sanctuary https://sheffield.cityofsanctuary.org/our-work/the-multiagency-drop-in/

### Acceptance criteria

Context is given: 
- do we know these people?
- how is the relationship managed?
- should any be in signposts?

### Owner

n/a this system doesn't work for our team

### Stakeholder

@emcglackern1

### Approver

@SallyMcGrath

**Comment by AmyCYF** [comment]
@SallyMcGrath the only one I currently work with is REUK.

**Context:**

-Do we know these people?
 - Last July I emailed REUK asking if they would be interested in hearing about the DLP and share the course with their community. They are currently referral partners (share our course details with their communities) and they are hoping to host on our cohorts in the second half of the year.

How is the relationship managed?
 - I email Niwaeli when course marketing material is shared. We recently met up at the Hillsong venue when I went to do a site inspection.

Should any be in signposts?
 - I don't do any signposting currently, but I could initiate conversations on potential sign posts to REUK?

Regarding the question of should we work with them - I am happy to reach out to these organisation and have calls about the program and see if there is potential for collaboration.
